tests/integration/test_collator.py

```diff
@@ -57,3 +62,56 @@ def it_knows_the_display_order_for_a_dimension(
         display_order = PayloadOrderCollator.display_order(dimension_, ())
 
         assert display_order == expected_value
+
+
+class DescribeSortByValueCollator(object):
+    """Partial-integration test suite for `SortByValueCollator` object."""
+
+    @pytest.mark.xfail(reason="WIP", raises=NotImplementedError, strict=True)
+    @pytest.mark.parametrize(
+        "order, xtop, xbot, element_vals, empty_idxs, expected_value",
```

Comment:
I can't figure the meaning of `xtop` by just reading this test.

Reply:
I figured since these values appear in the `"exclude": {"top": xtop, "bottom": xbot}` structure in the test body, that I could manage to get these parameters to fit on a single line and keep the test shorter. I don't generally try to make the tests explain the concepts of the code they exercise, reserving that effort for the docstrings in the code itself. But I can think about that more for future tests.